src/releng_docs/deploy/regular.rst

```diff
@@ -83,6 +131,7 @@ Protocal that we follow is:
 
    .. todo:: need to explain how to revert when a deployment goes bad.
 
+TODO: we can already do this while waiting for the release to happen
 #. Fill in the release notes on GitHub
```

Comment:
TODO: Move this entire section above "Verify that all production projects are now deployed", and say "While the production deployment happens, you can fill in the release notes on GitHub".

TODO: Link to https://github.com/mozilla/release-services/releases